frontends/mit-open/src/pages/UnitsListingPage/UnitsListingPage.tsx

```diff
@@ -424,9 +424,12 @@ const UnitsListingPage: React.FC = () => {
             <PageHeaderContainerInner>
               <PageHeaderText>
                 MIT is dedicated to advancing knowledge beyond students enrolled
-                in MIT's campus programs. Several units within MIT offer
+                in MIT’s campus programs. Several units within MIT offer
```

Comment:
I would leave as-is. We use normal apostrophes most places.
```suggestion
                in MIT's campus programs. Several units within MIT offer
```